exonum-java-binding/pom.xml

```diff
@@ -178,6 +179,12 @@
         <scope>provided</scope>
       </dependency>
 
+      <dependency>
```

Comment:
do we really need `pf4j` at the parent module?

Reply:
It is in `pluginManagement` — is not a dependency unless declared in sub modules, and it *will* be used in multiple modules (each service needs it to access `@Extension`).

Reply:
Are you also saying "Why doesn't pf4j come in several artifacts (e.g., pf4j-annotations, pf4j-core)?" That's a good question. Sawing it might be non-trivial and is certainly is a breaky.